gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -151,7 +151,7 @@ fn v1_routes(gateway: Gateway) -> Router {
         .route(GET_GATEWAY_ID_ENDPOINT, get(get_gateway_id))
         // These routes are for next generation lightning
         .route(ROUTING_INFO_V2_ENDPOINT, post(routing_info_v2))
-        .route(PAY_BOLT11_INVOICE_V2_ENDPOINT, post(pay_bolt11_invoice_v2))
+        .route(SEND_PAYMENT_V2_ENDPOINT, post(pay_bolt11_invoice_v2))
```

Comment:
Remind me, because I feel like you recently changed this from `SEND_PAYMENT_V2_ENDPOINT` to `PAY_BOLT11_INVOICE_V2_ENDPOINT`? Perhaps we should have separate endpoints for bolt11 and bolt12, to give us some flexibility going forward?

Reply:
Yes, however I found a rather clean way to process the LightningInvoice enum in the endpoint so switched it to send_payment to potentially support bolt11 and bolt12 through the same endpoint as much of the logic is shared.